### PR TITLE
fixed 14 character issues in parent screen in  completed

### PR DESCRIPTION
**Pull request by RakeshReddyVura**
Completed fixed 14 character issues parent screen

### CODE REVIEW

**Comment by RakeshReddyVura** [comment]
reopened the PR

**Review by bhanugummana — APPROVED** [review]

**Comment by RakeshReddyVura** [comment]
Added the same logic for other pages

File: src/pages/Leaderboard.tsx [hunk 1]
```diff
@@ -226,7 +226,7 @@ const Leaderboard: React.FC = () => {
               }
               alt=""
             />
-            {currentStudent?.name}
+            <span id="currentStudent-name">{currentStudent?.name}</span>
```
Comment (Sutara-Skanda) [inline src/pages/Leaderboard.tsx:229]: Follow the naming convention for ids for everywhere like this **#page-name-element-appropriate-name**

File: src/pages/Leaderboard.css [hunk 2]
```diff
@@ -50,3 +50,10 @@
 #leaderboard-avatar {
   padding: 5% 0%;
 }
+
+#currentStudent-name{
```
Comment (Sutara-Skanda) [inline src/pages/Leaderboard.css:54]: Follow the naming convention for ids for everywhere like this #page-name-element-appropriate-name

**Review by Sutara-Skanda — CHANGES_REQUESTED** [review]

**Comment by RakeshReddyVura** [comment]
Updated the logic with the naming convention

**Review by Sutara-Skanda — APPROVED** [review]

**Pull request merged**